lib/internal/test_runner/tests_stream.js

```diff
@@ -18,6 +18,7 @@ class TestsStream extends Readable {
       objectMode: true,
       highWaterMark: NumberMAX_SAFE_INTEGER,
     });
+    this.streams = [];
```

Comment:
This is introducing new public API that I don't think we want to support.

Reply:
I can redo it with a symbol / _? Unless you know a better way?

Reply:
Technically, it doesn't need to be attached to the stream at all. For example, the `reporterScope` object is internal and could hold them (you would need to make sure it works with watch mode though).

Reply:
IMO attaching to the stream makes the most sense, it's the.. well... stream.